subprojects/platform-jvm/src/main/java/org/gradle/jvm/toolchain/internal/DefaultToolchainSpec.java

```diff
@@ -107,6 +108,14 @@ public boolean isConfigured() {
 
     @Override
     public boolean isValid() {
+        if (vendor.getOrNull() == JvmVendorSpec.IBM_SEMERU) {
```

Comment:
:x: This will force realize the property early. We should find a way to validate this much later.